addons/web/static/src/core/ui/ui_service.js

```diff
@@ -30,10 +30,16 @@ export function useActiveElement(refName) {
     const uiService = useService("ui");
     const owner = useRef(refName);
 
-    let lastTabableEl, firstTabableEl;
-
     function trapFocus(e) {
-        switch (getActiveHotkey(e)) {
+        const hotkey = getActiveHotkey(e);
+        if (!["tab", "shift+tab"].includes(hotkey)) {
+            return;
+        }
+        const el = e.currentTarget;
+        const tabableEls = getTabableElements(el);
+        const firstTabableEl = tabableEls[0] || el;
+        const lastTabableEl = tabableEls[tabableEls.length - 1] || el;
```

Comment:
Didn't knew that accessing an index that don't exists just returns `undefined` 🤯